ClamavPlugin.php

```diff
@@ -106,14 +115,12 @@ function getActions($request, $verb) {
 	function manage($args, $request) {
 		switch ($request->getUserVar('verb')) {
 			case 'settings':
-				$contextID = (!is_null($this->_getBackwardsCompatibleContext()) ? $this->_getBackwardsCompatibleContext()->getId() : CONTEXT_SITE);
+				$contextID = (!is_null($this->_getBackwardsCompatibleContext()) ? $this->_getBackwardsCompatibleContext()->getId() : PKPApplication::CONTEXT_SITE);
```

Comment:
Now that `_getBackwardsCompatibleContext()` isn't doing any backwards-compatible work, it can be removed.